internal/buildinfo/version_test.go

```diff
@@ -25,5 +25,5 @@ func TestDevMeta(t *testing.T) {
 
 	loadMeta()
 
-	assert.Equal(t, fmt.Sprintf("0.0.0-%d+dev", BuildDate().Unix()), Version().String())
+	assert.Equal(t, fmt.Sprintf("0.0.0-%d+dev", BuildDate().Unix()), ParsedVersion().String())
```

Comment:
This is going to start expecting a calver, based on the time of the test. not sure the best way to make this easily testable...

Reply:
Going to mark this as resolved since (I think) I modified the code to prefer the version var if it parses, and that seems to be set in the testcase. Could be that gremlin rules apply and we shouldn't run this test straddling midnight, but we can cross that bridge if we get there and it actually exists. :)